### DISCUSSION TITLE
[Improvement]Add possibility to navigate to user page

### DISCUSSION THREAD

**Discussion by pandafliesinspace** [holypower777/global-chat#5] [Ideas]
precondition:
specific chat history is opened

As a USER I want to navigate to the streamer's page to view his details and chat history. Add hyperlink or button into the red square zone
![image](https://user-images.githubusercontent.com/26408366/166452889-3486f1a0-f708-49e8-ad4d-01f40a60e9d6.png)

**Comment by holypower777** [comment]
Done, now its clickable and redirects to message history of that user!
